src/frontend/src/lib/api/idb.api.ts

```diff
@@ -1,21 +1,25 @@
 import { browser } from '$app/environment';
+import { ETHEREUM_NETWORK_SYMBOL } from '$env/networks.env';
 import type { IdbEthAddress } from '$lib/types/idb';
 import type { Principal } from '@dfinity/principal';
 import { isNullish } from '@dfinity/utils';
 import { createStore, del, get, set, update, type UseStore } from 'idb-keyval';
 
 // There is no IndexedDB in SSG. Since this initialization occurs at the module's root, SvelteKit would encounter an error during the dapp bundling process, specifically a "ReferenceError [Error]: indexedDB is not defined". Therefore, the object for bundling on NodeJS side.
-const oisyEthAddressesStore = browser
-	? createStore('oisy-eth-addresses', 'eth-addresses')
-	: ({} as unknown as UseStore);
+export const idbAddressesStore = (identifier: string) =>
```

Comment:
nth:

- would you mind rename to `key` or something like that? `identitfier` automatically means `account identifier` for me. old habits.

- not sure, the function is `void` or returns a type?

Reply:
right! done both!